docs/src/administration/users.md

```diff
@@ -1,138 +1,162 @@
+Each environment type groups one or more environments together so that you can manage access for all environments of a certain type.
+Platform.sh offers three types of environment groups: Production, Staging, and Development.
+You can assign user permissions for each environment type.
+Any permissions you assign to an environment type apply to all environments of that type.
 
-{{< note >}}
-The project owner - the person licensed to use Platform.sh - has the project administrator role. The project owner is the only user that can make changes to the project plan. 
-{{< /note >}}
+For example, if you assign user1 **Admin** permissions for Development environments, User1 effectively has **Admin** permissions to all development environments.
 
-## SSH access
+A few things to consider:
 
-By default, everyone with access equal or greater than **Contributor** can access the project through SSH. 
+* Only one environment per project can be the Production type. It's set automatically as the default branch and can't be overridden separately.
+* You can change an environment type and have multiple Staging or Development environments.
+* To customize who can use SSH, [set the access key](/configuration/app/access.html) in your platform.app.yaml
 
-To customize who can use SSH, [set the `access` key](/configuration/app/access.md) in your `.platform.app.yaml` file.
 
-## Manage user permissions in the console
+| Environment permission   | Description |
+|------------------------- |-------------|
+|Viewer                    | Viewers can view all environments of this type.|
+|Contributor               | Contributors can push code and branch all environments of this type. This role includes the Viewer permissions.
+|Admin                     | Admins can change settings and execute actions on all environments of this type. This role includes the Viewer and Contributor permissions.
 
-From your list of projects, select the project where you want to view or edit user permissions. Click the Settings tab at the top of the page, then click the `Access` tab on the left to see the users in the project and their roles.
+## Manage users
 
-![Project user management screenshot](/images/management-console/settings-project-access.png)
+### Add a user to a project
 
-Selecting a user to edit that user's permissions or delete the user's access to the project entirely.
+To add a user to a project, follow these steps:
 
-To add a new user, click the `Add` button.
+{{< codetabs >}}
+---
+title="In the console"
+---
+1. Go to your console and select the project where you want to add a new user.
+2. Under **Settings**, click **Access**.
+1. Click **+ Add**
+1. Add the user's details and choose their permissions.
+4. Click **Save**.
+5. After you add a user from a project or an environment type, you must trigger a redeploy to propagate SSH access changes to each environment. You can redeploy by clicking **Redeploy** in the console.
```

Comment:
Given that the explanation is the same for both console and CLI, I'd consider putting it after the outcome and then just having something like: by either clicking **Redeploy in the console or running `platform redeploy`

That would help avoid problems like not having this idea in the CLI currently.

```suggestion
After you add a user to a project, you have to redeploy each environment for the SSH access changes to apply there.
```